Remove OrderedMap export and use SignaturesMap

mcintyre94 opened the pull request:
Not sure what I was doing here, but OrderedMap is just used for SignaturesMap, and all the uses were just using it as that

mcintyre94 commented:
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2518?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2518** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2518?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈
* **#2517** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2517?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#2516** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2516?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#2515** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2515?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#2514** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2514?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#2513** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2513?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#2494** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2494?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#2493** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2493?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#2492** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2492?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#2487** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2487?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>: 1 other dependent PR ([#2488](https://github.com/solana-labs/solana-web3.js/pull/2488) <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2488?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>)
* **#2486** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2486?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#2485** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2485?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#2484** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2484?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#2481** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2481?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @mcintyre94 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="11px" height="11px"/> <b>Graphite</b></a>
  <!-- Current dependencies on/for this PR: -->

lorisleiva reviewed (approved):
The title says "Remove OrderedMap export" but I can't see an actual export statement or keyword being removed here. Just flagging in case that wasn't intentional.

steveluscher reviewed (approved):
![Added via Giphy](https://media3.giphy.com/media/12ZDIx1Mw1cXVm/giphy.gif?cid=c9487743wf0z9ni8qo8dhier3heuas1qr02jbo8dl1x1cyxx&ep=v1_gifs_search&rid=giphy.gif&ct=g)

mcintyre94 commented:
> The title says "Remove OrderedMap export" but I can't see an actual export statement or keyword being removed here. Just flagging in case that wasn't intentional.

TY! Looks like I just removed the usages and forgot to remove the export lol. Updated!

mcintyre94 commented:
### Merge activity

* **Apr 19, 12:17 AM EDT**: @mcintyre94 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2518).
